### ISSUE TITLE
Matplotlib exporter does not work on Python 3

### ISSUE THREAD

**Issue by gmarull** [pyqtgraph/pyqtgraph#574]
Hi,

When trying to export a plot using the Matplotlib exporter, the following error is raised:

```
...
  File "/usr/local/lib/python3.5/dist-packages/pyqtgraph/exporters/Matplotlib.py", line 46, in cleanAxes
    for loc, spine in ax.spines.iteritems():
AttributeError: 'dict' object has no attribute 'iteritems'
```

Apparently it is because iteritems is being used (Python 2.x only).

**Comment by Kishore96in** [comment]
To fix this bug, we need to change
`for loc, spine in ax.spines.items()` to `for loc, spine in list(ax.spines.items())` in line 46 of pyqtgraph/exporters/Matplotlib.py. After making this change, the export works properly.

**Comment by Kishore96in** [comment]
Sorry, it seems the bug is already fixed. I was checking with the latest release, which is apparently more than a year old.

**Comment by ojob** [comment]
Seems to me this is a duplicate of #162.

**Comment by ixjlyons** [comment]
This has been fixed in #690
